lms/djangoapps/teams/teams_features.py

```diff
@@ -1,124 +1,41 @@
-import logging
-
-from django.conf import settings
-from web_fragments.fragment import Fragment
-
+"""Methods for teams modifications"""
 from courseware.courses import get_course_by_id
-from courseware.model_data import FieldDataCache
-from courseware.module_render import get_module_for_descriptor
-from openedx.core.djangoapps.crawlers.models import CrawlersConfig
 from student.models import CourseEnrollmentManager
 from opaque_keys.edx.locator import BlockUsageLocator
-
-from xmodule.x_module import STUDENT_VIEW
+from opaque_keys import InvalidKeyError
 
 CONTENT_DEPTH = 2
-SECTION_NAME = getattr(settings, "TEAM_SECTION", "Teams")
-LOG = logging.getLogger(__name__)
-
-
-class ModifyTeams(object):
-    """this class alows to modifies the team's behavior"""
-
-    def __init__(self, request, user, course_key):
 
-        self.effective_user = user
+class TeamsFeatures(object):
```

Comment:
I would call the class "CourseTeamsFeatures", for me it gives the context this is for a course